src/universal-dapp.js

```diff
@@ -53,30 +53,6 @@ UniversalDApp.prototype.getAccounts = function (cb) {
     }
 };
 
-UniversalDApp.prototype.getBalance = function (address, cb) {
```

Comment:
I won't have time. The account system on the UI side must be reworked. Currently we have a dropdown of accounts available and the one selected is used to make the next transaction. My idea was to somehow make that nicer and display a balance next to each address.

Either update after every transaction or have an *update balance* button.